website/docs/reference/widgets/table/README.md

```diff
@@ -103,6 +103,30 @@ This method uses a value in the API response as the key to the following API cal
 
 <img src="/img/pagination_(2)_(2).gif" width="70%" ></img>
 
+#### Meta properties based pagination
+
+Table widget exposes two meta properties, `previousPageVisited` and `nextPageVisited`. These values of these properties are set when a user moves to a new page. Depending on the direction of the new visited page with respect to the current page, the values of these properties are set.
```

Comment:
These values of these -> The value of these

Reply:
Fixed